<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shop</class>
 <widget class="QDialog" name="shop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>商店</string>
  </property>
  <widget class="QLabel" name="lblWhitPig">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBlackPig">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHuaZhu">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnWhiteBuy">
      <property name="text">
       <string>购买白猪</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBlackBuy">
      <property name="text">
       <string>购买黑猪</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHuaBuy">
      <property name="text">
       <string>购买花猪</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelWhiteIntroduction">
      <property name="text">
       <string>白猪:￥100/只</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelBlackIntroduction">
      <property name="text">
       <string>黑猪:￥100/只</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelHuaZhuIntroduction">
      <property name="text">
       <string>花猪:￥100/只</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
